source/portable/BufferManagement/BufferAllocation_2.c

```diff
@@ -64,6 +64,7 @@
 #endif /* ipconfigUSE_TCP == 1 */
 
 /*_RB_ This is too complex not to have an explanation. */
+/* coverity[misra_c_2012_rule_20_10_violation] */
```

Comment:
Consider making this a project wide exclusion.

Reply:
it would make similar macro additions go unnoticed